NOTICE.md

```diff
@@ -22,10 +26,11 @@ SPDX-License-Identifier: Apache-2.0
 
 ## Source Code
 
-The project maintains the following source code repositories 
-in the GitHub organization https://github.com/catenax-ng:
+The project maintains the following source code repositories
+in the GitHub organization https://github.com/eclipse-tractusx:
 
-* https://github.com/catenax-ng/product-DAPS
+* https://github.com/eclipse-tractusx/<my_product_repo1>
```

Comment:
add your repo here and remove the other example line.

Reply:
Added the repo & removed example line
Also removed .DS_STORE file